coriolisclient/base.py

```diff
@@ -38,6 +41,17 @@ def getid(obj, possible_fields=["uuid", "id"]):
     return obj
 
 
+def catch_unauthorized_exception(func):
```

Comment:
In my mind 'catch' = suppresses (i.e. does NOT re-raise the exception), so a more fitting name would be 'wrap_unauthoriezed_exception'